app/(auth)/welcome/index.tsx

```diff
@@ -1,11 +1,53 @@
-import { Link } from 'expo-router'
-import { Text, View } from 'react-native'
+import { useState } from 'react'
+import { View } from 'react-native'
+import PagerView from 'react-native-pager-view'
+import { ButtonLink, CarouselIndicators, Welcome } from '../../../components'
+
+const url = '../../../assets/images/'
+
+const imageSource = {
+  primarySource: require(`${url}welcome_background.webp`),
+  secondarySource: require(`${url}welcome_background_3.webp`),
+  thirdSource: require(`${url}welcome_background_2.webp`)
+}
+
+const carouselSection = [
```

Comment:
cambiar nombre de variable a una mas declarativa, en su lugar : 
```suggestion
const pageList = [
 {
    key: 0,
    title: 'Welcome to Mova',
    description: 'The best movie streaming app of the century to make your days great',
    image: imageSource
  },
]
```